src/status_im/wallet_connect_legacy/core.cljs

```diff
@@ -142,14 +142,27 @@
                     (:wallet-connect-legacy/sessions db))]
     {:wc-1-clean-up-sessions connectors}))
 
+(fx/defn save-session
+  {:events [:wallet-connect-legacy/save-session]}
+  [{:keys [db]} {:keys [peer-id dapp-name dapp-url connector]}]
+  (let [info (.stringify js/JSON (.-session connector))]
+    {:db db
```

Comment:
you don't need a db if there is no changes

Reply:
Done